.github/settings.yml

```diff
@@ -0,0 +1,34 @@
+# Collaborators: give specific users access to this repository.
+# See https://docs.github.com/en/rest/reference/repos#add-a-repository-collaborator for available options
+collaborators:
+  # Maintainers
+  - username: mmlb
+    permission: maintain
+  - username: micahhausler
+    permission: maintain
```

Comment:
Serious question, do we want to alphabetize by GitHub username?

Reply:
heh actually alphabetizing is why I looked at the one-line-per to begin with :smil